flytekit/remote/remote.py

```diff
@@ -1892,9 +1892,12 @@ def _get_input_literal_map(self, execution_data: ExecutionDataResponse) -> liter
             with self.remote_context() as ctx:
                 tmp_name = os.path.join(ctx.file_access.local_sandbox_dir, "inputs.pb")
                 ctx.file_access.get_data(execution_data.inputs.url, tmp_name)
-                return literal_models.LiteralMap.from_flyte_idl(
-                    utils.load_proto_from_file(literals_pb2.LiteralMap, tmp_name)
-                )
+                proto_type, value = utils.load_one_proto_from_file(tmp_name, literals_pb2.InputData,
```

Comment:
interesting that we had to only change here,
did you update the getData method in flyteadmin, if so, this will need an additional if
https://github.com/flyteorg/flytekit/blob/452a2ecde0bf4f9ddd1ed25dc8b446f1ad0e3e7d/flytekit/remote/remote.py#L273